packages/next-adapter/package.json

```diff
@@ -51,16 +51,17 @@
     "rimraf": "^3.0.2"
   },
   "peerDependencies": {
-    "next": "^9",
-    "react": "^16",
-    "react-native-web": "^0.13.12"
+    "next": "^11",
+    "react": "^17",
```

Comment:
Is React 17 compatible with Expo / RN 0.63?

Reply:
I haven't tried before this morning but yeah looks like it's working. I asked myself the same question because it seems React Native landed the support for React 17 in RN 0.64 🤷‍♂️

Reply:
I know this was jumping the gun and pushing RN 0.64 before Expo SDK, but this PR is a big push to get the next-adaptor updated to all the major versions and includes some future proofing.

However updating to React 17 did cause issues between NextJS and babel. In NextJS the JSX transform is [automatically enabled when using React 17](https://nextjs.org/docs/upgrading#react-16-to-17) and I'm not sure it can be disabled. Choosing between staying on React 16 or enabling the transform, I opted for the transform.

https://github.com/expo/expo-cli/pull/3586/files#diff-304e49a338415a9ceeba16c106fcbd2225cebddb9731f172988365509c7788dbL27

@axeldelafosse I personally haven't had any issues with this, but I envision there are edge cases where some Expo users may need to enable `useTransformReactJsxExperimental` for all platforms.